src/utils/boostUtils.js

```diff
@@ -0,0 +1,53 @@
+export const defaultBoostValue = 500
+export const minBoostValue = 0
+export const maxBoostValue = 1000
```

Comment:
I'm curious what this max should be. 1,000 OGN could be a lot or a little. We only have 200 OGN in our test accounts. @matthewliu do you have an opinion on this?

Reply:
To be clear, I don't think we should block merging on this comment. I think we should figure out the max and make any necessary changes in another PR.

Reply:
@cuongdo we've talked about making it dynamic based on the price and/or type of listing but I don't know that we've settled on an algorithm yet. I just set it to 1,000 for testing so that we could verify the "you don't have enough token" functionality with 200 in the wallet.